src/CplexSolverInterface.jl

```diff
@@ -341,18 +342,18 @@ export cbaddboundbranchup!,
 @compat abstract type CplexCallbackData <: MathProgCallbackData end
 
 # set to nothing to clear callback
-setlazycallback!(m::CplexMathProgModel,f) = (m.lazycb = f)
-setcutcallback!(m::CplexMathProgModel,f) = (m.cutcb = f)
-setheuristiccallback!(m::CplexMathProgModel,f) = (m.heuristiccb = f)
+MathProgBase.setlazycallback!(m::CplexMathProgModel,f) = (m.lazycb = f)
+MathProgBase.setcutcallback!(m::CplexMathProgModel,f) = (m.cutcb = f)
+MathProgBase.setheuristiccallback!(m::CplexMathProgModel,f) = (m.heuristiccb = f)
 setbranchcallback!(m::CplexMathProgModel,f) = (m.branchcb = f)
 setincumbentcallback!(m::CplexMathProgModel,f) = (m.incumbentcb = f)
-setinfocallback!(m::CplexMathProgModel,f) = (m.infocb = f)
+MathProgBase.setinfocallback!(m::CplexMathProgModel,f) = (m.infocb = f)
 
 function cbgetmipsolution(d::CplexCallbackData)
```

Comment:
The the `cb` functions also need a `MathProgBase.` prefix if they're implementing the MPB API.